README.md

```diff
@@ -3,11 +3,11 @@
 `sumologic-otel-lambda` publishes preconfigured [OpenTelemetry Lambda](https://github.com/open-telemetry/opentelemetry-lambda) layers which provide instrumentation for AWS Lambda functions.
 Released `sumologic-otel-lambda` layers are available:
 
-- Java wrapper layer contains OpenTelemetry Java `v1.24.0` and OpenTelemetry Collector `v0.75.0`. Please see list of [lambda layers](./java/README.md).
+- Java wrapper layer contains OpenTelemetry Java `v1.24.0` and OpenTelemetry Collector `v0.75.0`. Please see list of [lambda layers](https://github.com/SumoLogic/sumologic-otel-lambda/blob/release-java-v1.24.0/java/README.md).
 
-- NodeJS layer contains OpenTelemetry JavaScript SDK `v1.12.0` and OpenTelemetry Collector `v0.75.0`. Please see list of [lambda layers](./nodejs/README.md).
+- NodeJS layer contains OpenTelemetry JavaScript SDK `v1.12.0` and OpenTelemetry Collector `v0.75.0`. Please see list of [lambda layers](https://github.com/SumoLogic/sumologic-otel-lambda/blob/release-nodejs-v1.12.0/nodejs/README.md).
 
-- Python layer contains OpenTelemetry Python SDK `v1.17.0` with instrumentation `v0.38b0` and OpenTelemetry Collector `v0.75.0`. Please see list of [lambda layers](./python/README.md).
+- Python layer contains OpenTelemetry Python SDK `v1.17.0` with instrumentation `v0.38b0` and OpenTelemetry Collector `v0.75.0`. Please see list of [lambda layers](https://github.com/SumoLogic/sumologic-otel-lambda/blob/release-python-v1.17.0/python/README.md).
```

Comment:
How are you going to remember about updating the links if needed?

Reply:
There is a note in releasing guide about updates :)